<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #000000;
}
QWidget#centralwidget {
    background-color: #000000;
}
QLineEdit {
    background-color: #000000;
    color: white;
    border: none;
    padding: 20px;
    font-size: 48px;
    font-family: -apple-system, BlinkMacSystemFont, 'SF Pro Text', Arial;
    font-weight: 300;
}
QPushButton {
    background-color: #333333;
    color: white;
    border: none;
    border-radius: 50%;
    font-size: 24px;
    font-family: -apple-system, BlinkMacSystemFont, 'SF Pro Text', Arial;
    min-width: 65px;
    min-height: 65px;
    margin: 4px;
}
QPushButton:hover {
    background-color: #4d4d4d;
}
QPushButton:pressed {
    background-color: #666666;
}
QPushButton#btnClear, QPushButton#btnBackspace {
    background-color: #a5a5a5;
    color: #000000;
}
QPushButton#btnClear:hover, QPushButton#btnBackspace:hover {
    background-color: #d4d4d4;
}
QPushButton#btnDivide, QPushButton#btnMultiply, QPushButton#btnMinus, QPushButton#btnPlus {
    background-color: #ff9f0a;
    color: white;
}
QPushButton#btnDivide:hover, QPushButton#btnMultiply:hover, QPushButton#btnMinus:hover, QPushButton#btnPlus:hover {
    background-color: #ffb340;
}
QPushButton#btnEquals {
    background-color: #ff9f0a;
    color: white;
}
QPushButton#btnEquals:hover {
    background-color: #ffb340;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLineEdit" name="display">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnBackspace">
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDivide">
        <property name="text">
         <string>÷</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnMultiply">
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnMinus">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn5">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnPlus">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" rowspan="2">
       <widget class="QPushButton" name="btnEquals">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="btn0">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnDot">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
